bigstore/bigstore.py

```diff
@@ -339,14 +357,19 @@ def request_s3_credentials():
     print
     print "Enter your Amazon S3 Credentials"
     print
+    s3_bucket = raw_input("Bucket Name: ")
     s3_key = raw_input("Access Key: ")
     s3_secret = raw_input("Secret Key: ")
-    s3_bucket = raw_input("Bucket Name: ")
+    s3_profile_name = raw_input("Profile Name: ")
```

Comment:
In your AMI you might want to setup a profile name instead of using the default one. This might allow you to use i.e multi-factor authentication instead of using the secret and key setup here.